In [1]:
%matplotlib inline


import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

# Configure libraries
# The seaborn library makes plots look nicer
sns.set()
sns.set_context('talk')

# Don't display too many rows/cols of DataFrames
pd.options.display.max_rows = 7
pd.options.display.max_columns = 8

# Round decimals when displaying DataFrames
pd.set_option('precision', 2)

In [9]:
df = pd.read_csv("designer.csv")
df = df[['Job Title', 'Salary Estimate', 'Industry', 'Sector','Location']]
df

Job Title               Salary Estimate  \
0                        Product Designer  $129K-$153K (Glassdoor est.)   
1         Product Designer: Subscriptions  $129K-$153K (Glassdoor est.)   
2          Fire Sprinkler System Designer  $129K-$153K (Glassdoor est.)   
..                                    ...                           ...   
997      Senior Product Design/UX Manager   $94K-$187K (Glassdoor est.)   
998       Senior User Experience Designer   $94K-$187K (Glassdoor est.)   
999  Experience Designer (UX, UI, Visual)   $94K-$187K (Glassdoor est.)   

                                    Industry  \
0                                   Internet   
1                                 Publishing   
2                               Construction   
..                                       ...   
997       Home Furniture & Housewares Stores   
998  Enterprise Software & Network Solutions   
999                              IT Services   

                                 Sector           Location  
0                Information Technology  Mountain View, CA  
1                                 Media     Washington, DC  
2    Construction, Repair & Maintenance       Rockford, IL  
..                                  ...                ...  
997                              Retail         Boston, MA  
998              Information Technology        Houston, TX  
999              Information Technology         Denver, CO  

[1000 rows x 5 columns]

In [33]:
location = df['Location'].value_counts()
location

San Francisco, CA    110
New York, NY          82
Los Angeles, CA       63
                    ... 
Hilliard, OH           1
Bothell, WA            1
Gainesville, FL        1
Name: Location, Length: 194, dtype: int64

It is shown that SF has 110 of the listings, New York has 82 and LA has 63 of the listings at 